module-cheatsheets/scRNA-seq-advanced-cheatsheet.md

```diff
@@ -167,8 +169,93 @@ Read the full documentation and download cheatsheets (where available) for these
 
 
 
-### `Seurat` and `SCE` object conversion
+## `Seurat` and `SCE` object conversion
+
+When converting between `Seurat` and `SCE` objects, it's helpful to know how the different object types store and refer to similar information.
+The table below shows different aspects of single-cell objects and how to access the associated data, assuming the default names for each type of single-cell object.
+Importantly, these default names may not be the same - for example, `SCE` refers to normalized counts as `"logcounts"`, but `Seurat` refers to normalized counts as `"data"`.
+The table below additionally assumes that the `Seurat` object assay of interest is named `"RNA"`, which is the default assay name that `Seurat` assigns.
+
+| Data aspect | `SCE` | `Seurat` |
+|------------|---------|---------|
+| Raw counts assay | `counts(sce_object)` | `seurat_obj[["RNA"]]@counts` |
+| Normalized counts assay | `logcounts(sce_object)` | `seurat_obj[["RNA"]]@data` |
+| Reduced dimension: PCA matrix | `reducedDim(sce_object, "PCA)` | `seurat_obj$pca@cell.embeddings` |
+| Reduced dimension: UMAP matrix | `reducedDim(sce_object, "UMAP)` | `seurat_obj$umap@cell.embeddings` |
+| Cell-level metadata | `colData(sce_object)` | `seurat_obj@meta.data` |
+| Feature (gene)-level metadata | `rowData(sce_object)` | `seurat_obj[["RNA"]]@meta.features`|
+| Miscellaneous additional metadata | `metadata(sce_object)` | `seurat_obj@misc`|
```

Comment:
👏🏼 👏🏼 👏🏼 

This I think will be really useful! (My only fear is that it will make people ask more for the "Seurat equivalent" for all the things.)

Reply:
If anything it's useful for me!